contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -79,6 +85,7 @@ def _transmit(self, envelopes):
                 headers=headers,
                 timeout=self.options.timeout,
                 proxies=json.loads(self.options.proxies),
+                allow_redirects=False,
```

Comment:
Is requests library already handling redirects?, any reason to not just use that?

Reply:
We need to change the endpoint to the new location and `requests` library does not supply that if we let it handle redirects.